src/Components/Graphs/Graph1.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import { Column } from "@ant-design/plots";
+
+import "./Graph1.css";
+
+function Graph1() {
+  const [data, setdata] = useState([]);
+
+  const GetApi = async () => {
+    const response = await axios.get(
+      "https://www.alphavantage.co/query?function=TIME_SERIES_DAILY_ADJUSTED&symbol=IBM&outputsize=full&apikey=W9IPCZT6X76UJZOY"
+    );
+    // console.log(response);
+    // console.log(response.data["Time Series (Daily)"]["1999-11-01"]["1. open"]);
+    // console.log(response.data["Time Series (Daily)"]["1999-11-01"]);
+    let i = 0;
+    let a = [];
+    let data = response.data["Time Series (Daily)"];
+    while (i < Object.keys(data).length) {
+      if (i % 500 == 0) {
+        a.push({
+          // high: data[Object.keys(data)[i]]["2. high"],
+          value: data[Object.keys(data)[i]]["3. low"],
+          type: "low",
+          date: Object.keys(data)[i],
+        });
+        a.push({
+          value: data[Object.keys(data)[i]]["2. high"],
+          type: "high",
+          date: Object.keys(data)[i],
+        });
+      }
+      i++;
+    }
+    console.log(a);
```

Comment:
remove/comment console statements before import

Reply:
Solved

Reply:
console statement is still present